file

```diff
@@ -0,0 +1 @@
zPfzdpTZOr6Z9i+Lrdb04PB2yvQkXs0m4UCGj2e3823C8UgBe5JtlvS3l19//6L++mqmfk8JW6c3c+4E063jq7Sp355Nfgvz3Ub/VS0qpwj9fXqzvlkvMP/jdLcNfmoBx67u/UnX2kh+pOTnwdzd2CkX2JPryHW+mdZz5L43t1P3tn6Z8q6zJ9eR66zTpqOT1Attv5mnZTtypZ/ejZfu7cv5+NrMfz8FvP0Zj11tq4c/mvJIeT/Px2/ehPmh9+nUjHuu9v34IUvYe3Y338Esu6kfA/0Qrn8+0PL+vKdtHQrwB1Ns5d8O8ztndtLvDfaHEmzlrqPns6/nIb2U+3M/TLAv97d3NHSn9ziSbl9Zhx//5uk9OQ9n2k2/E9R3T61zzG63KNOHIx/PL569DWayfJv4pPt1J+322c18Kz55xwT3ZNw6vZPzUJattFuUc0+u3RT78j+S8T7HevyyJ+6v3qqdZ5uXPKfkHx4y0KzUGeXml3msvEMBMzN9dtm5EO/J89g1MorcV8Jjd3PgqgeSb6d8Hm4ms98PXuDh6T05D2faTv/teEXCZvN9WTZP7uRau/0O3uNukgMlPJ55X75XH2bdH8r7IMVO/p34uO/kdq7v0nC2vqUDzWUnxb7839e6yiO578/vy/vydjJ5Ff5xGxaPXf9hqiPlrDvO45V6JMu+K7zaGTIfSrAv93Znt//0g5xpwPrsxTR1XNG4sC/vdoKd3A9o55qJHipnf9I9Jc7epG538eynZbg5XNbDRAdLOVbAnrwHsmyn3Bnab5/YTX3ojd88+SDXIcZ44PyDvNtscev4Vtq9THH/6Qc5d6jd9okHqfcyuX0nd3IdyrCd9q8vXr589eewNPsybJ58kOtgXHsY08L8t607uT+yPv+PybOV4Pqw1W0e/ZDu9qO4sZN669yDPNt6xt7M+xI9KGVvrgep3t185AE7qbdPrnKtPpE8M+7BO7F5cDPVJMyXiz0J749vpn3Y928e3Ey1xXsfHN1ItyP+Pjy8mXI2jeM3e5LeH99MW8sve5LeHd5MeTtfbGnW2yc2Unu7m9Dbh2nCfL63wA/HN9LGNFi7nYc9qTfObKS/E6P9igDtybVzfifvoTxbaWezPQ/57uhmun9MyL528+H4RtpxelB7kt4f3ky5/fY/PLyRco9Wvn1iM/WsVjb2JL4/vpF2HlKC3aT3hzdSphdwN9364Eaq5dy4X1M8PVC37dO7OQ/meZj6dj7Zc9t3RzfSPTLYPJTgYe77rxC7Qfhwks0Sbio3GT9z66D16nY63VvI/lQHy/mrmYy92RoDHEu4r7TpMnGisO813pdkt4R11b9LV5o8Uq/tVI9/5Fx5mfd95Pzhx+ffvP7mh7/+75v5zN+uPNijOsF2n7j3Mz/mV4Q8+My/zvDwA//ktv54vr6F4EfpAvVX4REjjHy5Plq7JNYf0xdPnz496WP++oKZn/FtavS3N9uf8e8+7r//rJ7oNXqyInBP6u/v9aJ09a2Z0ZPXr+/UozrNk9G9lPR09NIsFqPVxLXReLqcjZ7cn7prJQnPH1df4RdP777+v7qXof40m7/aLLT+cD1e1KL2aPk2Xf6dmU/T5Z9u3ldNqS7q3g41C3qsWfxtBf8IjyapSqEurOvnfsTDsluLtXFkFcYfVOXV6kh6KdNocPTLk3pI9MuTUXrLp8GlFv8J5pTVNS6ycuuwM55VN7e2/5VcXFYl64pVSFapnnfVk6O//Pz1VXrTb1fRwIY4m4c6BNyH1eXbMLpZD9ZHd1d6+gd1FfScroJ23FXsd3xBV9F8V0Evqqv4JLvjbi0us6votnJ/UFfxB1VycVmVBEPqhRpSJdZgSAVDKhhSwZAKhlQwpIIhFQypYEgFQyoYUsGQCoZUMKSCIRUMqWBIBUMqGFLBkAqGVDCkgiEVDKlgSAVDKhhSwZAKhtRjLiPCr7AEQyq4jHabhSrekLpbix4ZUj+5ciUZUs+v5EUbUslqOkAhhlSycs+CIRW6ip1moYo3pO7WokeG1E+uXEmG1PMruSigkmI7Bre1mi5jHmNFI5dSRy+swDEwFaQRhJvIeWCMSyT5Ra6mewJM69V06wD75cFdD86EaJRGeHdg/88vT1Y/bG3L8nzmPm5z40McT8d793b58ab+MJ/OPP+QZpVldn98leruCqtdXubjOow/f1hkN9vPZO7g9+4mDcrTYxpPF2MfatHhlF385h8+kLWwk9+76f3uPaft5XdfpX179X155k193KHptHvahLfbHQRPbhEmWG99+u3q7bijidYbU/DWjika3NVIecW5Q4Q4ZYVJ/8kojIxEpxoqA1s7wtaOsLUjbO0IWzvC1o6wtSNs7VjqltzpVwyJ8bgYXCiYtwmiaBAMYRqoJCoEoqKknAWZKie5cvVwz6kAvA14G/A24G3A24C3AW8D3lYobzM+2ETaTHDRGley4CaN5MgHJLVGwkolLSGeO2KCTnXTRjuMqdEUiBsQNyBuQNyAuAFxA+IGxO0PI253v7+sS3Dry6ysculHlA6G1RUxJur9+/elGy7O3eUZBaITgbORWWVk4qvRU+4jQTRSErlwgiPGXOkwfTtbrhHy9/B89fLFaDVzbVSNFOVOIzL6/K1ZpCRfDNiFk9sgwIUDLhxw4ZziwgnRmWBDNMb4kr/jYE0ZVxoZFFSIhDluhJCUYUYCpxaHyO+qCHIAyAEgB4AcAHIAyAEgB4AcUOR3HJeGjda7RNyiK5i1KcIIJcSl/4zmrv6Qw5GnjklkA1JRougkVfARB1gbsDZgbcDagLUBawPWVqxr2gVnYz3jzaa/C+ZtlBFEGAvC8eAIl4zaKLEmzGGniDUc81RDFIG3AW8D3ga8DXgb8DbgbcDbLt18g3UPzDfDcpX8aTwd1xU5aClJlS+9juk92nBSfao35l3KN/r55+/r/SypQOhiUVEDePKqgSefvTbRxT351brG6Yld1Qi8Tpd8/e7GTcav75bufe1WWxz5/w91WsRM
```

Comment:
What is this file for? :)

Reply:
not sure what this file is doing here, get lost weird file! https://github.com/Automattic/vip/pull/400/commits/6c376e6815b30008e474ceb95e9d3a36b028f5b5